src/test/kotlin/com/cjbooms/fabrikt/generators/MicronautAuthenticationTest.kt

```diff
@@ -0,0 +1,203 @@
+package com.cjbooms.fabrikt.generators
```

Comment:
It would be good to see these tests either replaced with or complemented with the existing style of test where code examples are included in the resources folder.
I know it's a bit more cumbersome, but it allows the test cases to form a living documentation that can quickly be consulted to see what code gets generated for what API spec

Reply:
Good point. We have added an example directory and added the tests to the general generator tests.